src/App.js

```diff
@@ -1,28 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import Content from './templates/Content';
+import theme from './utils/scss/theme';
 
 function App() {
-  const { data, isLoading } = useFeaturedBanners();
-  console.log(data, isLoading);
+  const page = 'homepage';
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <ThemeProvider theme={theme}>
+      <Content page={page} />
```

Comment:
I saw the `Content` component renders the Homepage, my recommendation is to remove it and call from here directly the `Homepage`, I don't see much value to having the `Content` component but if you have good points to leave, please reply. Lastly, move the homepage component to a `pages` directory due to we are creating a web app so we are rendering pages.

<img width="1358" alt="Screen Shot 2022-04-21 at 20 19 33" src="https://user-images.githubusercontent.com/96157153/164578195-2011ecff-9da7-4005-97da-5e135afdd0ef.png">